src/Admin/IdeasWorkshopThemeAdmin.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace AppBundle\Admin;
+
+use AppBundle\IdeasWorkshop\ThemeManager;
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+
+class IdeasWorkshopThemeAdmin extends AbstractAdmin
+{
+    protected $datagridValues = [
+        '_page' => 1,
+        '_per_page' => 32,
+        '_sort_order' => 'ASC',
+        '_sort_by' => 'name',
+    ];
+
+    private $themeManager;
+
+    public function __construct(
+        string $code,
+        string $class,
+        string $baseControllerName,
+        ThemeManager $themeManager
+    ) {
+        parent::__construct($code, $class, $baseControllerName);
+
+        $this->themeManager = $themeManager;
```

Comment:
Instead of override the construct to inject the manager, you can inject it by setter injection.
You can create a `setThemeManager` method and configure the call method :  

```
<service id="app.admin.ideas_workshop_theme" class="AppBundle\Admin\IdeasWorkshopThemeAdmin">
     <tag name="sonata.admin" manager-type="orm" label="Thèmes des notes" group="Atelier des idées" />
     <argument />
     <argument>AppBundle\Entity\IdeasWorkshop\Theme</argument>
     <argument />

    <call method="setThemeManager">
        <argument type="service" id="AppBundle\IdeasWorkshop\ThemeManager"/>
    </call>
</service>
```